tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -470,7 +470,8 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
           val content = JsObject("extra" -> "read all about it".toJson, "yummy" -> true.toJson)
           val p = if (path.endsWith("/")) "/" else ""
           invocationsAllowed += 1
-          m(s"$testRoutePath/$path", content) ~> Route.seal(routes(creds)) ~> check {
+          m(s"$testRoutePath/$path", HttpEntity(ContentTypes.`application/json`, content.compactPrint)) ~> Route.seal(
```

Comment:
Tried to get rid of `compactPrint`, could not get the tests to compile by just passing the `JsObject`.

Reply:
why do we need to set the content type explicitly on the entity?